lib/Parse/ParseDecl.cpp

```diff
@@ -5487,89 +5478,68 @@ Parser::parseDeclOperator(ParseDeclOptions Flags, DeclAttributes &Attributes) {
 }
 
 ParserResult<OperatorDecl>
-Parser::parseDeclPrefixOperator(SourceLoc OperatorLoc, Identifier Name,
+Parser::parseDeclOperatorImpl(SourceLoc OperatorLoc, Identifier Name,
                                 SourceLoc NameLoc, DeclAttributes &Attributes) {
-  SourceLoc lBraceLoc;
-  if (consumeIf(tok::l_brace, lBraceLoc)) {
-    auto Diag = diagnose(lBraceLoc, diag::deprecated_operator_body);
-    if (Tok.is(tok::r_brace)) {
-      SourceLoc lastGoodLocEnd = Lexer::getLocForEndOfToken(SourceMgr,
-                                                            NameLoc);
-      SourceLoc rBraceEnd = Lexer::getLocForEndOfToken(SourceMgr, Tok.getLoc());
-      Diag.fixItRemoveChars(lastGoodLocEnd, rBraceEnd);
-    }
-
-    skipUntilDeclRBrace();
-    (void) consumeIf(tok::r_brace);
-  }
+  bool isPrefix = Attributes.hasAttribute<PrefixAttr>();
+  bool isInfix = Attributes.hasAttribute<InfixAttr>();
+  bool isPostfix = Attributes.hasAttribute<PostfixAttr>();
```

Comment:
In this PR or later, I suggest to diagnose multiple fixture here.
Currently, `postfix infix operator ++++` is accepted.

Although `prefix postfix` combinations are [diagnosed in parseNewDeclAttribute](https://github.com/jtbandes/swift/blob/d6590cd781c6389dae4514ac0784f778d08da206/lib/Parse/ParseDecl.cpp#L310-L315),
we should not simply add `DAK_Infix` there, because that will emit superfluous diagnostic for `func` decl:
```
infix prefix func ++(x: Int, y: Int) {}
error: attribute 'infix' cannot be combined with this attribute
error: 'infix' modifier is not required or allowed on func declarations
```